contracts/governance/extensions/GovernorVotesQuorumFraction.sol

```diff
@@ -38,7 +36,7 @@ abstract contract GovernorVotesQuorumFraction is GovernorVotes {
      * @dev Returns the current quorum numerator. See {quorumDenominator}.
      */
     function quorumNumerator() public view virtual returns (uint256) {
-        return _quorumNumeratorHistory._checkpoints.length == 0 ? _quorumNumerator : _quorumNumeratorHistory.latest();
+        return _quorumNumeratorHistory.latest();
```

Comment:
I don't understand this change, what I see is that if `_quorumNumeratorHistory.checkpoints.length == 0`, we used to return `_quorumNumerator`, but now we return 0 for that case. 

Are we sure it's equivalent?

Reply:
The previous code was there to return a correct value for a contract that was upgraded from a prior version, which only had `_quorumNumerator` and not `_quorumNumeratorHistory`, to this newer version. In that case, we wanted to return the previous value if there were no new checkpoints in the History.

This can be removed now because no contracts will be upgraded to 5.0 from a previous version, so `_quorumNumerator` will never be set, in fact it will always be 0 so the code is equivalent.

Reply:
Also, if you don't upgrade to this, you can expect the constructor to be called, and the constructor push a value, making length is > 0